relaton/serializers/bibxml/reference.py

```diff
@@ -127,32 +132,26 @@ def create_reference(item: BibliographicItem) -> Element:
     else:
         ref.set('anchor', anchor)
 
-    # refcontent
-    refcontent = create_refcontent(item.extent)
-    if refcontent:
-        ref.set("refcontent", f"{refcontent}")
-
     return ref
 
 
-def create_refcontent(extent: Union[LocalityStack, Locality]) -> Union[None, str]:
-    refcontent = None
+def create_refcontent(extent: Union[LocalityStack, Locality, None]) -> Union[None, str]:
```

Comment:
I recommend using `Optional[...]` instead of `Union[..., None]`, I’m moving codebase to that.

And instead of `Union[..., ...]` we should be able to use the new `... | ...` annotation shorthand.

Reply:
See updated signature: https://github.com/relaton/relaton-py/commit/c9121673457280f69dda29ed4943188c964d8430#diff-fc6c843ab71694a81afbf85fd6f7fdeebbdacbc6a219803f7eba800c900d3ff8L138-R145